src/server/src/main/java/io/cassandrareaper/management/jmx/JmxCassandraManagementProxy.java

```diff
@@ -367,7 +367,7 @@ public Map<String, String> getEndpointToHostId() {
   }
 
   @Override
-  public String getPartitioner() {
+  public String getPartitioner() throws ReaperException {
     Preconditions.checkNotNull(ssProxy, "Looks like the proxy is not connected");
```

Comment:
No, since the `StorageServiceMbean.getPartitionerName()` signature doesn't throw anything, we can't properly get the errors.
I'll wrap this in a try/catch block for RuntimeExceptions and wrap it in a ReaperException.

Reply:
Sounds good. I'll approve now and you can merge once you've made this change and all checks are passing.